packages/components/organisms/f-checkout/src/components/Error.vue

```diff
@@ -22,23 +21,74 @@
             data-test-id="checkout-error-page-description">
             {{ $t(`errorMessages.${errorFormType}.description`) }}
         </p>
+
+        <f-button
+            :class="$style['c-checkout-error-button']"
+            button-size="large"
+            button-type="primary"
+            is-full-width
+            data-test-id="error-page-redirect-to-menu-button"
+            @click.native="redirectToMenu">
+            {{ $t(`errorMessages.${errorFormType}.buttonText`) }}
+        </f-button>
     </card>
 </template>
 
 <script>
+import { mapState } from 'vuex';
 import Card from '@justeat/f-card';
+import FButton from '@justeat/f-button';
 import SadBagIconDecorator from '../assets/images/jet-sad-bag.svg';
+import '@justeat/f-button/dist/f-button.css';
 import '@justeat/f-card/dist/f-card.css';
+import { VUEX_CHECKOUT_MODULE } from '../constants';
+import loggerMixin from '../mixins/logger.mixin';
 
 export default {
     components: {
         Card,
+        FButton,
         SadBagIconDecorator
     },
+    mixins: [
+        loggerMixin
+    ],
     props: {
         errorFormType: {
             type: String,
             required: true
+        },
+        restaurantMenuPageUrl: {
+            type: String,
+            default: ''
+        }
+    },
+
+    computed: {
+        ...mapState(VUEX_CHECKOUT_MODULE, [
+            'basket'
+        ])
+    },
+
+    mounted () {
+        if (this.$logger) {
```

Comment:
For some reason the tests were throwing errors so added this in. Do we use the logger in the mounted hook in checkout elsewhere?

Reply:
Not sure if we do, but it should be available on `mounted`, so I'd say it might be a missing mock or similar in one of the tests.

Reply:
Did add a mock for $logger [here](https://github.com/justeat/fozzie-components/pull/1034/files#diff-77d6d5e41e52cc7565e75f71e027e67a362fae9f1f6e0c8954c6b95ea57f8d50R45)

Reply:
Any updates on this? Because I don't think we should be adding these validations because of the tests. We should see why the tests are failing/throwing console errors.

Reply:
Fixed now. Didn't realise the failure was coming from checkout.vue instead of error.vue. 👍